config/pmix.m4

```diff
@@ -945,14 +944,15 @@ AC_DEFINE_UNQUOTED([PMIX_ENABLE_DSTORE],
                  [if want shared memory dstore feature])
 
 #
+#
```

Comment:
There is a `if test "$enable_dstore" == "no" ; then` just above this line that was missed.